lib/pod/command/dependencies.rb

```diff
@@ -38,14 +38,8 @@ def validate!
         if @podspec_name
           require 'pathname'
           path = Pathname.new(@podspec_name)
-          if path.exist?
-            @podspec = Specification.from_file(path)
-          else
-            @podspec = SourcesManager.
```

Comment:
This was not working at all so I deleted it. It might be better to add in another PR.

So one can do `pod SomeRemote.podspec` and it gets pulled instead of being present locally.